composer.json

```diff
@@ -32,9 +38,9 @@
         "ext-xml": "*",
         "ext-zip": "*",
         "composer-plugin-api": "^2.1",
-        "browscap/browscap": "^6.0",
-        "browscap/browscap-php": "^5.0",
-        "cache/filesystem-adapter": "^1.1",
+        "browscap/browscap": "6.0.x-dev",
```

Comment:
This will need updating once the upstream release is made; likely we'll be looking to release this as `6.1.0` actually